lib/doh.c

```diff
@@ -173,6 +183,9 @@ UNITTEST DOHcode doh_encode(const char *host,
   return DOH_OK;
 }
 
+#define doh_encode(host, dnstype, dnsp, len, olen) \
+  (doh_encodepfx(host, NULL, dnstype, dnsp, len, olen))
```

Comment:
Thanks for the review; it helps my sense of "curl culture". "Old function name": fine. "Until there's a user": also fine. Looking at the ROADMAP, ESNI seems to be the only potential user currently on the radar, and that needs IETF to reach a certain stage with SVCB/HTTPSSVC, so we needn't hold our breath.

Your call whether to leave this PR here so that it's still warm when needed, or close it and start another whenever.